### PR TITLE
NAS-109934 / 21.04 / Install Docker image submit issue / NAS-109934

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

**Review by ryousaf — APPROVED** [review]
LGTM

File: src/app/pages/applications/forms/chart-release-add.component.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class ChartReleaseAddComponent implements OnDestroy {
   protected queryCallOption: Array<any>;
   protected addCall: string = 'chart.release.create';
   protected isEntity: boolean = true;
-
+  protected isLinear: boolean = true;
```
Comment (undsoft) [inline src/app/pages/applications/forms/chart-release-add.component.ts:29]: Is it used?

**Review by undsoft — COMMENTED** [review]
There was also this PR: https://github.com/truenas/webui/pull/5291/files
If PRs are equivalent, I'd suggest we approve and merge PR from a contributor – he was first and it may help build a better community.

**Comment by AlexKarpov98** [comment]
@undsoft Fixed

File: src/app/pages/applications/forms/chart-release-add.component.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/applications/forms/chart-release-add.component.ts]: Good catch!

**Pull request closed without merging**